argocd/README.md

```diff
@@ -109,6 +109,39 @@ spec:
 # (...)
 ```
 
+##### Troubleshooting 
+
+**Clashing Tag Names**:
+The Argo CD integration attaches a name tag derived from the application name OpenMetrics label when available. This could sometimes lead to querying issues if a name tag is already attached to a host, as seen in the example `name: host_a, app_a`. To prevent any unwanted behavior when querying, it is advisable to [remap the name label][13] to something more unique, such as argocd_app_name if the host happens to already have a name tag. Example configuration:
```

Comment:
```suggestion
The Argo CD integration attaches a name tag derived from the application name OpenMetrics label when available. This could sometimes lead to querying issues if a name tag is already attached to a host, as seen in the example `name: host_a, app_a`. To prevent any unwanted behavior when querying, it is advisable to [remap the name label][13] to something more unique, such as `argocd_app_name` if the host happens to already have a name tag. Below is an example configuration:
```